<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Digital Sculptor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>File name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditFileName">
        <property name="text">
         <string>Sculpt</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="60,2,36,2">
      <item>
       <widget class="SculptorDisplay" name="sculptorDisplay" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>13</width>
          <height>13</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_9">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Adjusts</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="2" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Z size</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QSlider" name="horizontalSliderYRadius">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QSlider" name="horizontalSliderRadius">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="3" column="2">
             <widget class="QLCDNumber" name="lcdNumber_8">
              <property name="segmentStyle">
               <enum>QLCDNumber::Flat</enum>
              </property>
              <property name="intValue" stdset="0">
               <number>1</number>
              </property>
             </widget>
            </item>
            <item row="6" column="1">
             <widget class="QSlider" name="horizontalSliderZRadius">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLCDNumber" name="lcdNumber">
              <property name="digitCount">
               <number>5</number>
              </property>
              <property name="segmentStyle">
               <enum>QLCDNumber::Flat</enum>
              </property>
              <property name="intValue" stdset="0">
               <number>1</number>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Radius</string>
              </property>
             </widget>
            </item>
            <item row="5" column="2">
             <widget class="QLCDNumber" name="lcdNumber_4">
              <property name="segmentStyle">
               <enum>QLCDNumber::Flat</enum>
              </property>
              <property name="intValue" stdset="0">
               <number>1</number>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Y radius</string>
              </property>
             </widget>
            </item>
            <item row="4" column="2">
             <widget class="QLCDNumber" name="lcdNumber_3">
              <property name="segmentStyle">
               <enum>QLCDNumber::Flat</enum>
              </property>
              <property name="intValue" stdset="0">
               <number>1</number>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QLCDNumber" name="lcdNumber_7">
              <property name="segmentStyle">
               <enum>QLCDNumber::Flat</enum>
              </property>
              <property name="intValue" stdset="0">
               <number>1</number>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>X size</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Y size</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSlider" name="horizontalSliderYSize">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>X radius</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QSlider" name="horizontalSliderXRadius">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QSlider" name="horizontalSliderZSize">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSlider" name="horizontalSliderXSize">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="6" column="2">
             <widget class="QLCDNumber" name="lcdNumber_5">
              <property name="segmentStyle">
               <enum>QLCDNumber::Flat</enum>
              </property>
              <property name="intValue" stdset="0">
               <number>1</number>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLCDNumber" name="lcdNumber_2">
              <property name="segmentStyle">
               <enum>QLCDNumber::Flat</enum>
              </property>
              <property name="intValue" stdset="0">
               <number>1</number>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Z radius</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Z Plan</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QSlider" name="horizontalSliderPlan">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLCDNumber" name="lcdNumber_6">
              <property name="segmentStyle">
               <enum>QLCDNumber::Flat</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_11">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Views</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="2" column="1">
             <widget class="QPushButton" name="pushButtonBottomView">
              <property name="text">
               <string>Bottom View</string>
              </property>
              <property name="iconSize">
               <size>
                <width>50</width>
                <height>50</height>
               </size>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="pushButtonFrontView">
              <property name="text">
               <string>Front View</string>
              </property>
              <property name="iconSize">
               <size>
                <width>50</width>
                <height>50</height>
               </size>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="pushButtonTopView">
              <property name="text">
               <string>Top View</string>
              </property>
              <property name="iconSize">
               <size>
                <width>50</width>
                <height>50</height>
               </size>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QPushButton" name="pushButtonRightView">
              <property name="text">
               <string>Right View</string>
              </property>
              <property name="iconSize">
               <size>
                <width>50</width>
                <height>50</height>
               </size>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QPushButton" name="pushButtonBackView">
              <property name="text">
               <string>Back View</string>
              </property>
              <property name="iconSize">
               <size>
                <width>50</width>
                <height>50</height>
               </size>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QPushButton" name="pushButtonLeftView">
              <property name="text">
               <string>Left View</string>
              </property>
              <property name="iconSize">
               <size>
                <width>50</width>
                <height>50</height>
               </size>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QPushButton" name="pushButtonLayerView">
              <property name="text">
               <string>Layer View</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonColor">
            <property name="text">
             <string>Pick a color</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>13</width>
          <height>487</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNewModel"/>
   <addaction name="actionBuildOFF"/>
   <addaction name="actionPutVoxel"/>
   <addaction name="actionCutVoxel"/>
   <addaction name="actionPutBox"/>
   <addaction name="actionCutBox"/>
   <addaction name="actionPutSphere"/>
   <addaction name="actionCutSphere"/>
   <addaction name="actionPutEllipse"/>
   <addaction name="actionCutEllipse"/>
  </widget>
  <action name="actionNewModel">
   <property name="icon">
    <iconset resource="assets.qrc">
     <normaloff>:/icons/assets/newmodel.svg</normaloff>:/icons/assets/newmodel.svg</iconset>
   </property>
   <property name="text">
    <string>New Model</string>
   </property>
   <property name="toolTip">
    <string>New Model</string>
   </property>
  </action>
  <action name="actionBuildOFF">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="assets.qrc">
     <normaloff>:/icons/assets/buildoff.svg</normaloff>:/icons/assets/buildoff.svg</iconset>
   </property>
   <property name="text">
    <string>Build OFF</string>
   </property>
   <property name="toolTip">
    <string>Build OFF File</string>
   </property>
  </action>
  <action name="actionPutVoxel">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="assets.qrc">
     <normaloff>:/icons/assets/putvoxel.svg</normaloff>:/icons/assets/putvoxel.svg</iconset>
   </property>
   <property name="text">
    <string>Put Voxel</string>
   </property>
   <property name="toolTip">
    <string>Put Voxel</string>
   </property>
  </action>
  <action name="actionCutVoxel">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="assets.qrc">
     <normaloff>:/icons/assets/cutvoxel.svg</normaloff>:/icons/assets/cutvoxel.svg</iconset>
   </property>
   <property name="text">
    <string>Cut Voxel</string>
   </property>
   <property name="toolTip">
    <string>Cut Voxel</string>
   </property>
  </action>
  <action name="actionPutBox">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="assets.qrc">
     <normaloff>:/icons/assets/putbox.svg</normaloff>:/icons/assets/putbox.svg</iconset>
   </property>
   <property name="text">
    <string>Put Box</string>
   </property>
   <property name="toolTip">
    <string>Put Box</string>
   </property>
  </action>
  <action name="actionCutBox">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="assets.qrc">
     <normaloff>:/icons/assets/cutbox.svg</normaloff>:/icons/assets/cutbox.svg</iconset>
   </property>
   <property name="text">
    <string>Cut Box</string>
   </property>
   <property name="toolTip">
    <string>Cut Box</string>
   </property>
  </action>
  <action name="actionPutSphere">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="assets.qrc">
     <normaloff>:/icons/assets/putsphere.svg</normaloff>:/icons/assets/putsphere.svg</iconset>
   </property>
   <property name="text">
    <string>Put Sphere</string>
   </property>
   <property name="toolTip">
    <string>Put Sphere</string>
   </property>
  </action>
  <action name="actionCutSphere">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="assets.qrc">
     <normaloff>:/icons/assets/cutsphere.svg</normaloff>:/icons/assets/cutsphere.svg</iconset>
   </property>
   <property name="text">
    <string>Cut Sphere</string>
   </property>
   <property name="toolTip">
    <string>Cut Sphere</string>
   </property>
  </action>
  <action name="actionPutEllipse">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="assets.qrc">
     <normaloff>:/icons/assets/putellipse.svg</normaloff>:/icons/assets/putellipse.svg</iconset>
   </property>
   <property name="text">
    <string>Put Ellipse</string>
   </property>
   <property name="toolTip">
    <string>Put Ellipse</string>
   </property>
  </action>
  <action name="actionCutEllipse">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="assets.qrc">
     <normaloff>:/icons/assets/cutellipse.svg</normaloff>:/icons/assets/cutellipse.svg</iconset>
   </property>
   <property name="text">
    <string>Cut Ellipse</string>
   </property>
   <property name="toolTip">
    <string>Cut Ellipse</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SculptorDisplay</class>
   <extends>QWidget</extends>
   <header>sculptordisplay.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="assets.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>horizontalSliderXSize</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>918</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>988</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderYSize</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_2</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>918</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>988</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderXRadius</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_3</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>918</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>988</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderYRadius</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_4</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>918</x>
     <y>280</y>
    </hint>
    <hint type="destinationlabel">
     <x>988</x>
     <y>284</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderZRadius</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_5</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>918</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>988</x>
     <y>313</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderPlan</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_6</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>918</x>
     <y>425</y>
    </hint>
    <hint type="destinationlabel">
     <x>988</x>
     <y>429</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderZSize</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_7</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>888</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>937</x>
     <y>191</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderRadius</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_8</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>883</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>945</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
